src/js/views/createCertificate/View.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback, useState } from 'react';
+
+import { Box, Button } from '@material-ui/core';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+
+import { ViewContainer } from '../stateComponents';
+import { CONSTANTS } from './constants';
+import CollapsibleCard from './layout/CollapsibleCard';
+import useStyles from './style';
+
+const CreateCertificate = () => {
+  const { t } = useTranslation('createCertificate');
+  const history = useHistory();
+  const classes = useStyles();
+
+  const [expanded, setExpanded] = useState('');
+  const [isGeneratedCertificate, setIsGeneratedCertificate] = useState(false);
+
+  const toggleCollapseCard = useCallback(
+    clicked => {
+      if (expanded === clicked) {
+        setExpanded('');
+      } else {
+        setExpanded(clicked);
+      }
+    },
+    [expanded],
+  );
+
+  const handleLeaveCertificateCreation = () => {
+    if (history.length) history.goBack();
+    else history.push('/certificates');
+  };
+
+  return (
+    <ViewContainer headerTitle={t('Novo certificado')}>
+      <Box className={classes.container}>
+        <Box className={classes.content}>
+          {expanded !== CONSTANTS.CSR && expanded !== CONSTANTS.CA && (
+            <CollapsibleCard
+              expanded={expanded}
+              handleChange={toggleCollapseCard}
+              setIsGeneratedCertificate={setIsGeneratedCertificate}
+              isGeneratedCertificate={isGeneratedCertificate}
+              creationMethod={CONSTANTS.ONE_CLICK_CREATE}
+              title={t('createCertificateOneClick.title')}
+              subTitle={t('createCertificateOneClick.subTitle')}
+            />
+          )}
+
+          {expanded !== CONSTANTS.CA && (
```

Comment:
Sounds a bit confusing to me checking '!==', I know that is for making less code but is also less comprehensive.